src/docs/working-with-actors/creating-actors.md

```diff
@@ -2,39 +2,133 @@
 layout: docs.hbs
 title: Creating your first Actor
 ---
-# Creating your first Actor
-
->**Note**<br/>
+# Creating Actors
+> **Note**<br/>
 Since Akka.NET enforces parental supervision every actor is supervised and (potentially) the supervisor of its children, it is advisable that you familiarize yourself with [Actor Systems](Actor Systems) and [Supervision and Monitoring](Supervision) and it may also help to read [Actor References, Paths and Addresses](Addressing).
 
 ## Defining an Actor class
-Actors in C# are implemented by extending the `ReceiveActor` class and configuring what messages to receive using the `Receive<TMessage>` method.
+Actors are implemented by extending the `UntypedActor` class and implementing the `OnReceive` method. This method takes the message as a parameter.
```

Comment:
I dont agree with this. We want people to use the ReceiveActor. UntypedActor should only be used if your building your own actorbaseclass imho, or if you dont like the ReceiveActor api.

There used to be abit of confusion as to which Actor base class to use for normal actor implementations, and we have since taken on the stance that ReceiveActor should be used.

Some documentation still references the UntypedActor as an example. Bit as far as im concerned they should be changed to ReceiveActor as well.

@Aaronontheweb or @rogeralsing  could you comment on this?